### DISCUSSION TITLE
Tea

### DISCUSSION THREAD

**Discussion by khali245** [pkgxdev/pkgx#941] [Help]
How is this going to work?

**Comment by codewithneeraj** [comment]
Hi @khali245 

Here is the Quick Overview about tea protocol.

1. If you have any package deployed on (projects listed in APT, Crates, Homebrew, npm, pkgx, PyPI and RubyGems, with more to come in the future. ). then  your project may be qualifies to earn rewards on tea.

To check your Project eligibility follow these steps

GO to https://tea.xyz/rewards-for-oss-contributions
Type your package name in the search bar and check if it qualifies to earn rewards on tea.

>> Testnet will begin from 21st February where you will be able to check all the functionalities that tea provides.

You can also check this link for more info: https://tea.xyz/rewards-for-oss-contributions
